Appendix.Rmd

```diff
@@ -24,7 +24,7 @@ Several other university libraries have also created guides on data management y
 
 Feel free to remix these documents to suit your needs with credit to the NIH DMSP Guidance Working Group.
 
-In addition, there are several tools that can help you write your plan and make sure that it complies with all the policies that you need to comply with:
+In addition, there are several tools and resources that can help you write your plan and make sure that it complies with all the policies that you need to comply with:
```

Comment:
```suggestion
In addition, there are several tools and resources that can help you write your plan while complying with NIH policies:
```

Reply:
hmmm since some of them are not actually for writing I think "and" might be better - also for lots of policies beyond NIH

Reply:
I can still shorten it though

Reply:
how's that? I edited your suggestion @avahoffman